### PR TITLE
Update Alerts_qa.md

### PR DESCRIPTION
**Pull request by Men-hau**
QA sur la relation entre les dates des alertes, events et concentrateur

### CODE REVIEW

File: docs/xdr/FAQ/Alerts_qa.md [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+1- `log timestamp`:            Generated by the machine and present in the message (no actions done by SEKOIA.IO)
+2- `collector timestamp`:      Period when the log has been processed by the collector (timestamp collected from the endpoint available in the log unless it is missing so the timestamp of the header generated by the collector will be used instead)
+3- `event.created`:            Period when the log has been arrived in SEKOIA.IO
+4- `Alert date created`:       Period when the alert has been generated in SEKOIA.IO
+
+The chronological time should be as follow (from old to recent):
+`log timestamp (1) < collector timestamp (2) < event.created (3) < Alert created (4)`
+
+!!! Troubleshoot tips
+    Please check the different dates and collector configuration
```
Comment (ka0ula) [inline docs/xdr/FAQ/Alerts_qa.md:17]: is there a link to the collector configuration?

File: docs/xdr/FAQ/Alerts_qa.md [hunk 2]
```diff
@@ -1,7 +1,22 @@
-## Alert date
+## Alert date and additional events
 
 When an alert is triggered, additional events can enrich this alert but the date of the alert will not be updated (date= 1st trigger). 
 
+## Alert and events timestamp relation
+
+There are 4 informations available on SEKOIA.IO:
```
Comment (ka0ula) [inline docs/xdr/FAQ/Alerts_qa.md:7]:
```suggestion
There are 4 valuable information available on SEKOIA.IO:
```

File: docs/xdr/FAQ/Alerts_qa.md (hunk 1 above)
Comment (Men-hau) [reply docs/xdr/FAQ/Alerts_qa.md:17]: It is on client side when configuring their collector

**Comment by CharlesLR-sekoia** [comment]
@Men-hau the concentrator stage is optional as we are able to collect different types of logs from API or HTTP protocol. It only applies for OnPremise perimeter.

File: docs/xdr/FAQ/Alerts_qa.md (hunk 1 above)
Comment (Men-hau) [reply docs/xdr/FAQ/Alerts_qa.md:18]: @ka0ula From Charles comments, I have added as optional as it depends it only applies when a collector is set.

**Pull request closed without merging**